packages/lesswrong/lib/collections/books/collection.ts

```diff
@@ -12,7 +12,11 @@ export const Books: BooksCollection = createCollection({
 });
 
 export const makeEditableOptions = {
-  order: 20
+  order: 20,
+  getLocalStorageId: (book, name) => {
+    if (book._id) { return {id: `${book._id}_${name}`, verify: false} }
+    return {id: `collection: ${book.collectionId}_${name}`, verify: false}
```

Comment:
Are you sure about the `verify: false` here?

Reply:
Yeah, I think leaving it as true is better. Fixed.